src/javafundamentals/lambdas/Ex06ExceptionHandling.java

```diff
@@ -8,4 +10,26 @@
  */
 
 public class Ex06ExceptionHandling {
+    public static void main(String[] args) {
+        Squarer squarer = new Squarer();
+
+        try {
+            squarer.square(14, input -> {
+                if (input < (Integer.MAX_VALUE / Integer.MAX_VALUE)) {
```

Comment:
`(Integer.MAX_VALUE / Integer.MAX_VALUE) ` is equal to 1 , right?
So you are saying in the if:  `if (input < 1)`
I guess you just wanted to raise an exception but simplicity in the code matters!

Reply:
I was trying to do the square root but i clearly wasn't thinking about it.